src/index.html

```diff
@@ -3,8 +3,22 @@
 <head>
     <meta charset="UTF-8">
     <title>Country Information - World Regions</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+<script type="module" src="app.js"></script>
```

Comment:
Deze zou ik of onderaan je body zetten , of met een defer="" in de head-section.